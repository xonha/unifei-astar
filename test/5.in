10
0000000000
0000F00000
0F0000TT0T
000T00F0T0
00000T000T
0000000000
0000000000
0000000T00
T0T000F00T
0000T00000
