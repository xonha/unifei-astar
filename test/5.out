Tomou -120 de dano! (quanto mais negativo melhor)
